ios/ImagePickerManager.h

```diff
@@ -1,11 +1,12 @@
 #import <React/RCTBridgeModule.h>
 #import <UIKit/UIKit.h>
+#import <PhotosUI/PhotosUI.h>
 
 typedef NS_ENUM(NSInteger, RNImagePickerTarget) {
   camera = 1,
   library
 };
 
-@interface ImagePickerManager : NSObject <RCTBridgeModule, UINavigationControllerDelegate, UIActionSheetDelegate, UIImagePickerControllerDelegate>
+@interface ImagePickerManager : NSObject <RCTBridgeModule, UINavigationControllerDelegate, UIActionSheetDelegate, UIImagePickerControllerDelegate, PHPickerViewControllerDelegate>
```

Comment:
@ravirajn22 Out of curiosity, why do you prefer doing this on the public interface? For one, the outside world doesn't need to know that it conforms to the interface (personally I'd move `UIActionSheetDelegate`, `UIImagePickerControllerDelegate`, `UINavigationControllerDelegate` into the `.m` file too). This was one of the benefits that Obj-C has over Swift, that you can move the `#import` into the `.m` file making the link phase faster

Reply:
I am no expert in objective c, so l like consistency over others. Since all previous protocols have been done on the public interface as you said, I thought its better if we can declare PHPickerViewControllerDelegate the same way.

Refactors can go in another PR or with smaller PR, not with big PR like this.

Reply:
Understood! Then I'll gladly change this in a follow-up PR 👍